drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -286,10 +286,6 @@ msgid ""
 msgstr ""
 
 #: templates/html/js_templates.html
-msgid "Zones"
```

Comment:
@mtyaka yes, I've used xgettext to collect all the messages, but than manually added new ones to the file to avoid huge diff. And then fed `text.po` to msgfmt to build compiled files. I'll add the steps to README in a separate PR.